src/tpm.rs

```diff
@@ -33,6 +33,54 @@ ftpm_initialize.py
 Following are function from tpm_initialize.py program
 *****************************************************************/
 
+/*
+ * Return: Result wrap the execution result
+ *
+ * This function create a new ek if this hasn't been done before. Otherwise,
+ * it will give the error saying that ek has already been created. Same
+ * implementation as the original python-keylime version. This function only
+ * run once during the starting of cloud node.
+ */
+fn create_ek() -> Result<(), Box<String>> {
+    let (output, code, _) = run(
+        String::from("createek"),
+        EXIT_SUCCESS,
+        false,
+        false,
+        String::new(),
+    );
+
+    if let Some(EXIT_SUCCESS) = code {
+        Ok(())
```

Comment:
change this to a `return`, then de-indent the `else`.  Repeat below and the code should be much more flat.